### PR TITLE
Add missing released modules.

### PR DESCRIPTION
**Pull request by dereuromark**
Refs https://release.spryker.com/releases/view/474/CORE-3354-Supplier-management

### CODE REVIEW

**Comment by PhilinTv** [comment]
This is not a bug. It's a separate feature and it should be handled as a separte feature (epic). With QA and stuff. Please don't go around a process.

File: composer.json [hunk 1]
```diff
@@ -136,19 +136,26 @@
...
+    "spryker/company-business-unit-extension": "^1.0.0",
+    "spryker/company-business-unit-gui": "^1.0.0",
+    "spryker/company-business-unit-gui-extension": "^1.0.0",
     "spryker/company-data-import": "^0.1.0",
     "spryker/company-extension": "^1.0.0",
     "spryker/company-gui": "^1.0.0",
+    "spryker/company-gui-extension": "^1.0.0",
     "spryker/company-mail-connector": "^1.0.0",
     "spryker/company-role": "^1.0.0",
+    "spryker/company-supplier": "^1.0.0",
```
Comment (PhilinTv) [inline composer.json]: Supplier modules should not be wired to the suite (they confront upcoming Merchant concept).

Comment (Ole-Gi) [reply composer.json]: But what about company business unit?

Comment (PhilinTv) [reply composer.json]: Company Business Unit should be wired.

**Pull request closed without merging**